docs/modules/ROOT/pages/references/object-specifications.adoc

```diff
@@ -325,6 +337,52 @@ Settings:
 * `accessKeyIDSecretRef`: Kubernetes secret reference containing the the Access Key ID
 * `secretAccessKeySecretRef`: Kubernetes secret reference containing the Secret Access Key
 
+=== Azure
+
+Settings:
+
+* `container`: name of the container that should be used
+* `accountNameSecretRef`: Kubernetes secret reference containing the account name
+* `accountKeySecretRef`: Kubernetes secret reference containing the account key
+
+=== GCS
+
+Settings:
+
+* `projectIDSecretRef`: Kubernetes secret reference containing the Google project ID
+* `accessTokenSecretRef`: Kubernetes secret reference containing the Google access token
+* `bucket`: name of the bucket that should be used
+
+=== B2
+
+Settings:
+
+* `path`: Path of the B2 instance
+* `bucket`: name of the bucket that should be used
+* `accountIDSecretRef`: Kubernetes secret reference containing the account ID
+* `accountKeySecretRef`: Kubernetes secret reference containing the account key
+
+=== Local
+
+Settings:
+
+* `mountPath`: MountPath inside wrestic
```

Comment:
@Kidswiss is that right?

Reply:
Yes that should be right. Although the local backend was never really tested. It may need more fields in the future. For example to specify what it should mount into the container...

Reply:
Hm okay, since this is already "released" in 0.x, I think we should leave it for 1.x and do improvements/fixes in normal release schedule. I'd expect bug reports if something is missing here.